test/exercism/homework_test.rb

```diff
@@ -13,7 +13,9 @@ def test_status
     attributes = { user: alice, language: 'ruby' }
 
     UserExercise.create(attributes.merge(slug: 'leap', skipped_at: now))
-    UserExercise.create(attributes.merge(slug: 'clock', fetched_at: now))
+    UserExercise.create(attributes.merge(slug: 'clock', fetched_at: now, iteration_count: 0))
+    UserExercise.create(attributes.merge(slug: 'submitted_but_then_skipped', skipped_at: now, iteration_count: 1, fetched_at: now))
```

Comment:
This doesn't make sense. If you've submitted it, then it's not skipped. You'd only skip something so that you can fetch the next thing. If you've submitted it, you can fetch it.